frontend/javascripts/oxalis/view/left-border-tabs/mapping_settings_view.tsx

```diff
@@ -175,27 +176,36 @@ class MappingSettingsView extends React.Component<Props, State> {
       this.props.mapping &&
       this.props.hideUnmappedIds != null;
     const isDisabled = this.props.isEditableMappingActive || this.props.isMappingPinned;
+    const disabledMessage = this.props.isEditableMappingActive
+      ? "Editable mappings from proofreading actions cannot be disabled"
+      : this.props.mapping
+      ? "This mapping has been pinned to this annotation and thus cannot be disabled."
```

Comment:
```suggestion
      ? "The mapping has been pinned, because the segmentation was modified while it was active. It can no longer be disabled or changed."
```